index.html

```diff
@@ -15,305 +15,364 @@
-              feature that you need to built or a project that needs coding. I'd
-              love to help with it.
-            </p>
+          <div class="containerxx">
+            <div class="contact-title">
+              <h4>Contact me</h4>
+            </div>
+            <div class="contact-par">
+              <p>
+                If you have an application you are interested in developing, a
+                feature that you need to built or a project that needs coding.
+                I'd love to help with it.
+              </p>
+            </div>
+            <div class="form-container">
+              <form action="https://formspree.io/f/mleanrpl" method="post">
+                <ul class="list-form">
+                  <li class="list4">
+                    <input
+                      type="text"
+                      id="name"
+                      maxlength="30"
+                      name="user_name"
+                      placeholder="Name"
+                      required
+                    />
+                  </li>
+                  <li class="list4">
+                    <input
+                      type="email"
+                      id="mail"
+                      size="30"
+                      name="user_email"
+                      placeholder="email"
+                      required
+                    />
+                  </li>
+                </ul>
+                <textarea
+                  id="msg"
+                  name="user_message"
+                  placeholder="Write your message here"
+                  maxlength="500"
+                  required
+                ></textarea>
+                <button type="submit" class="btn">Get it Touch</button>
+              </form>
+            </div>
           </div>
-          <div class="form-container">
-            <form action="https://formspree.io/f/mleanrpl" method="post">
-              <ul class="list-form">
-                <li class="list4">
-                  <input
-                    type="text"
-                    id="name"
-                    maxlength="30"
-                    name="user_name"
-                    placeholder="Name"
-                    required
-                  />
-                </li>
-                <li class="list4">
-                  <input
-                    type="email"
-                    id="mail"
-                    size="30"
-                    name="user_email"
-                    placeholder="email"
-                    required
-                  />
-                </li>
-              </ul>
-              <textarea
-                id="msg"
-                name="user_message"
-                placeholder="Write your message here"
-                maxlength="500"
-                required
-              ></textarea>
-              <button type="submit" class="btn">Get it Touch</button>
-            </form>
+          <div class="barxx">
+            <div class="bar12"></div>
```

Comment:
- [ ] Remove this line, because the original design doesn't have one

**Your design:**

![image](https://user-images.githubusercontent.com/60631456/121700499-95608280-ca95-11eb-954c-288099ba208d.png)

**Original design:**

![image](https://user-images.githubusercontent.com/60631456/121700600-b032f700-ca95-11eb-8b03-d1137730e206.png)